tasks/engine_setup.yml

```diff
@@ -0,0 +1,65 @@
+---
+- block:
+  - name: Set answer file path
+    set_fact:
+      answer_file_path: "/tmp/answerfile-{{ lookup('pipe', 'date +%Y%m%d%H%M%SZ') }}.txt"
+
+
+  - name: Use the default answerfile
+    template:
+      src: answerfile_{{ ovirt_engine_setup_version }}_basic.txt.j2
+      dest: "{{ answer_file_path }}"
+      mode: 0600
+      owner: root
+      group: root
+    when: ovirt_engine_setup_answer_file_path is undefined
+
+  - name: Copy custom answer file
+    template:
+      src: "{{ ovirt_engine_setup_answer_file_path }}"
+      dest: "{{ answer_file_path }}"
+      mode: 0600
+      owner: root
+      group: root
+    when: ovirt_engine_setup_answer_file_path is defined
+
+  - name: Update setup packages
+    yum:
+      name: "ovirt*setup*"
+      state: latest
+    when: ovirt_engine_setup_update_setup_packages
+
+  - name: Update all packages
+    yum:
+      name: "*"
+      state: latest
+    when: ovirt_engine_setup_update_all_packages
+
+  - name: Set accept defaults parameter if variable is set
+    set_fact:
+      accept_defaults: "{{ '--accept-defaults' if ovirt_engine_setup_accept_defaults else '' }}"
+
+  - name: Run engine-setup with answerfile
+    command: "engine-setup --config-append={{ answer_file_path }} {{ accept_defaults }}"
+    tags:
+      - skip_ansible_lint
```

Comment:
Why do you need this? BTW, you can also set acceptdefaults in the answer file, no need to pass it on the command line. See also: https://bugzilla.redhat.com/1270719#8

Reply:
Nice one. Didn't know, I'll change.

Reply:
I tried to change to use this answerfile variable but it didn't work.. It failed on the question to auomatically upgrade the postgresql with EOF, which means the default answer was not passed. Moved back to command line option.